target/linux/ramips/patches-5.10/835-asoc-add-mt7620-support.patch

```diff
@@ -20,22 +20,14 @@ Signed-off-by: John Crispin <blogic@openwrt.org>
 
 --- a/arch/mips/ralink/of.c
 +++ b/arch/mips/ralink/of.c
-@@ -13,6 +13,7 @@
+@@ -14,6 +14,7 @@
  #include <linux/of_fdt.h>
  #include <linux/kernel.h>
  #include <linux/memblock.h>
 +#include <linux/module.h>
```

Comment:
I think this include is not needed any more, upstream now included ```#include <linux/export.h>```
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/diff/arch/mips/ralink/of.c?id=v5.10.42&id2=v5.10.41

Reply:
Indeed, I missed that. Will fix and update, thanks!

Reply:
Fixed, build-tested and updated.